virt/node_network/virt-updating-node-network-config.adoc

```diff
@@ -11,15 +11,31 @@ include::modules/virt-about-nmstate.adoc[leveloffset=+1]
 
 include::modules/virt-creating-interface-on-nodes.adoc[leveloffset=+1]
 
+[discrete]
```

Comment:
Just curious, what is the purpose of the [discrete] label?

Reply:
It prevents the title from showing in the TOC.
I used it here so that I could xref to the different policy examples after the early procedure where it's relevant, but having additional resources turn up as entry 3 in the TOC didn't look right (and those elements are already in the TOC, so it's only relevant to people who are looking at that particular procedure.